src/win/winapi.c

```diff
@@ -63,6 +64,10 @@ void uv_winapi_init(void) {
     uv_fatal_error(GetLastError(), "GetModuleHandleA");
   }
 
+  pRtlGetVersion = (sRtlGetVersion) GetProcAddress(
```

Comment:
It will fail with `ERROR_INVALID_PARAMETER` when you pass this flag to unsupported Windows.